src/d3d/nvapi_d3d_instance.h

```diff
@@ -3,22 +3,24 @@
 #include "../nvapi_private.h"
 #include "../resource_factory.h"
 
+constexpr static uint64_t kNanoInMicro = 1000;
```

Comment:
nit: for consistency and since we don't use this elsewhere, I would put this inside the private definition of the class, like we did here https://github.com/jp7677/dxvk-nvapi/blob/master/src/sysinfo/nvapi_adapter.h#L67

Reply:
Done